lib/new_relic/agent/instrumentation/view_component/instrumentation.rb

```diff
@@ -0,0 +1,37 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic::Agent::Instrumentation
+  module ViewComponent
+    INSTRUMENTATION_NAME = NewRelic::Agent.base_name(name)
+
+    def render_in_with_tracing(*args)
+      NewRelic::Agent.record_instrumentation_invocation(INSTRUMENTATION_NAME)
+
+      begin
+        segment = NewRelic::Agent::Tracer.start_segment(
+          name: metric_name(self.class.identifier, self.class.name)
+        )
```

Comment:
Normally I would expect a yield after this line, which would call `super/render_in_without_tracing` from the do blocks.

Reply:
Great catch @tannalynn. This wasn't yielding and preventing views from showing up 😱 Added the yield, tested, and we get both the NR metric and app view now.